engines/pink/gui.cpp

```diff
@@ -361,7 +363,10 @@ void PinkEngine::executeMenuCommand(uint id) {
 }
 
 void PinkEngine::openLocalWebPage(const Common::String &pageName) {
-	warning("Trying to open local webpage \"%s\", but webpage opening isn't implemented yet",pageName.c_str());
+	auto gameFolder= Common::FSNode(ConfMan.get("path"));
```

Comment:
We don't yet allow for 'auto' variables in the codebase. Please change these to explicit types

Reply:
I'm also concerned whether just sticking a file:// in front of the filename is portable across all browsers, particularly for Windows. What system did you test the code on?

Reply:
I test on Windows.

Reply:
That makes me a bit happier. Though according to
http://help.collab.net/index.jsp?topic=/faq/winletterdrive.html
The proper formatting for a Window path/filename is like the following:
file:///d:/some/path/to/repos/on/d/drive.txt

So just putting a file:// in front of the standard path/filename getPath() returns may not be portable for all possible browsers on Windows.